src/utils/error.ts

```
@@ -1,6 +1,6 @@
 import { MiddlewareHandler } from "hono";
 import { ContentfulStatusCode } from "hono/utils/http-status";
-import { AppOptions } from "index";
+import console from "console";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)